docs/DATAPART.md

```diff
@@ -0,0 +1,21 @@
+# DataPart
```

Comment:
#  Manually Maintain Mutable State

Also add 3 things:

1. Picture showing locking script is divided into Code & Data parts
2. sCrypt code sample on manual state: old counter.scrypt
3. test code sample on manual state: old counter.scrypttest.ts